fcli-core/fcli-ssc/src/main/java/com/fortify/cli/ssc/aviator/cli/cmd/SSCAviatorAuditCommand.java

```diff
@@ -0,0 +1,101 @@
+import com.fortify.cli.ssc.appversion.cli.mixin.SSCAppVersionResolverMixin;
+import com.fortify.cli.ssc.appversion.helper.SSCAppVersionDescriptor;
+import com.fortify.cli.ssc.appversion.helper.SSCAppVersionHelper;
+import com.fortify.cli.ssc.artifact.cli.mixin.SSCArtifactDownloadOptions;
+import kong.unirest.UnirestInstance;
+import lombok.Getter;
+import lombok.SneakyThrows;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import picocli.CommandLine.Mixin;
+import picocli.CommandLine.Option;
+import picocli.CommandLine.Command;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+
+@Command(name = "audit")
+public class SSCAviatorAuditCommand extends AbstractSSCJsonNodeOutputCommand implements IRecordTransformer, IActionCommandResultSupplier {
+    @Getter @Mixin private OutputHelperMixins.TableNoQuery outputHelper;
+    @Mixin private SSCAppVersionResolverMixin.RequiredOption appVersionResolver;
+    @Mixin private SSCArtifactDownloadOptions downloadOptions;
+    @Mixin private ProgressWriterFactoryMixin progressWriterFactory;
+    @Option(names = {"--tenant-token", "--at"}, required = true) private String tenantToken;
+    @Option(names = {"--tenant-name", "--tn"}, required = true) private String tenantName;
+    @Option(names = {"--aviator-url", "--au"}, required = true) private String aviatorUrl;
+    Logger logger = LoggerFactory.getLogger(SSCAviatorAuditCommand.class);
+
+    @Override
+    @SneakyThrows
+    public JsonNode getJsonNode(UnirestInstance unirest) {
+        try (var progressWriter = progressWriterFactory.create()) {
+            SSCAppVersionDescriptor av = appVersionResolver.getAppVersionDescriptor(unirest);
+            Path tempDir = Files.createTempDirectory("ssc_aviator_");
+
+            File fprFile;
+            if (downloadOptions.getDestination() != null && downloadOptions.getDestination().getFile() != null) {
+                fprFile = downloadOptions.getDestination().getFile();
+            } else {
+                fprFile = tempDir.resolve(String.format("%s_%s.fpr", av.getApplicationName(), av.getVersionName())).toFile();
+            }
+            SSCFileTransferHelper.download(
+                    unirest,
+                    SSCUrls.DOWNLOAD_CURRENT_FPR(av.getVersionId(), downloadOptions.isIncludeSources()),
+                    fprFile,
+                    SSCFileTransferHelper.ISSCAddDownloadTokenFunction.ROUTEPARAM_DOWNLOADTOKEN);
+
+            progressWriter.writeProgress("Status: Processing FPR");
+
+            var processedFile = AuditFPR.auditFpr(fprFile, tenantToken, tenantName, aviatorUrl);
+
+            progressWriter.writeProgress("Status: Uploading FPR to SSC");
+            JsonNode uploadResponse = uploadFpr(unirest, processedFile, av);
+            JsonNode dataNode = uploadResponse.get("data");
+            String id = dataNode.has("id") ? dataNode.get("id").asText() : "";
+
+            return av.asObjectNode()
```

Comment:
Maybe it would be good to return the same/similar data as 'fcli ssc artifact upload', or at least add an @DefaultVariablePropertyName annotation that points to the property containing the artifact id to either this class or SSCAviatorCommands (see SSCArtifactCommands), as this would allow for running something like the following to wait for processing of the audited FPR file to complete:

`fcli ssc aviator audit ... --store my_audit`
`fcli ssc artifact wait-for ::my_audit::`